src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/SecretBuildWrapper.java

```diff
@@ -71,6 +87,22 @@
             }
         };
     }
+    
+    @Override
+    public OutputStream decorateLogger(AbstractBuild build, OutputStream logger)
+    		throws IOException, InterruptedException, RunnerAbortedException {
+    	if(showBindings)
+    		return super.decorateLogger(build, logger);
+    	
+    	Map<String,String> overrides = new HashMap<String,String>();
+        List<MultiBinding.Unbinder> unbinders = new ArrayList<MultiBinding.Unbinder>();
```

Comment:
As referred in the next review comment, I thought it would be necessary to call the unbind step after calling the bind step, but that is not necessary anymore after changing the bind step to not do any actual binding, when called with an invalid launcher and workspaces (is null at this stage, but may change in future).